openstack/nova/templates/etc/_nova-policy.json.tpl

```diff
@@ -176,9 +177,9 @@
     "compute_extension:networks:view": "!",
     "compute_extension:networks_associate": "!",
     "compute_extension:os-tenant-networks": "!",
-    "compute_extension:quotas:show": "rule:context_is_viewer",
-    "compute_extension:quotas:update": "rule:context_is_admin",
-    "compute_extension:quotas:delete": "rule:context_is_admin",
+    "compute_extension:quotas:show": "rule:context_is_viewer or rule:context_is_quota_admin",
+    "compute_extension:quotas:update": "rule:context_is_quota_admin",
+    "compute_extension:quotas:delete": "rule:context_is_quota_admin",
     "compute_extension:quota_classes": "rule:context_is_admin",
```

Comment:
Why are `quota_classes` not necessary for `nova` but are necessary for `cinder`?

Reply:
The problem is that `quota_classes` (apparently) has only one permission for both read and write. I don't want to inadvertently take read permissions away from users that need it.